site/app/templates/forum/ThreadPostForm.twig

```diff
@@ -170,9 +170,11 @@
         </select>
         {% endif %}
         {% if show_announcement and core.getUser().accessFullGrading() %}
-            {% set announcement_text = email_enabled ? "Announcement?(w/ notification and email)" : "Announcement?(w/ notification)" %}
+            {% set announcement_text = email_enabled ? "Pin Thread?(w/email & notification)" : "Announcement?(w/ notification)" %}
             <label class="inline-block" for="Announcement">{{ announcement_text }}</label>
             <input type="checkbox" class="thread-announcement-checkbox" name="Announcement" value="Announcement" data-ays-ignore="true"/>
+            <label class="inline-block" for="Announcement">{{ "Pin Thread" }}</label>
+            <input type="checkbox" class="thread-announcement-checkbox" name="Announcement" value="Announcement" data-ays-ignore="false"/>
```

Comment:
Currently this checkbox still sends an email, even when the other one is not set, I think it works exactly the same as the checkbox above it. 
I do not know what `data-ays-ignore` does but I do not think that is what turns on or off the emails.